aiida/restapi/translator/calculation/__init__.py

```diff
@@ -38,3 +40,117 @@ def __init__(self, **kwargs):
         super(CalculationTranslator, self).__init__(
             Class=self.__class__, **kwargs)
 
+    @staticmethod
+    def get_retrieved_inputs(node, filename=None, rtype=None):
+        """
+        Get the retrieved input files for job calculation
+        :param node: aiida node
+        :return: the retrieved input files for job calculation
+        """
+
+        if node.dbnode.type.startswith("calculation.job."):
+
+            fullpath = os.path.join(node.get_abs_path(), "raw_input")
```

Comment:
Ideally, it's better not to use the path and then access files directly (this will go away soon with e.g. the support of swift). Instead, get `.folder`, then you can use `.get_subfolder(subfolder_name)` and then there are methods of `Folder` to get the list of files, get the files content, ... - if they are not enough we can add methods there, but we should avoid using direct methods accessing files

Reply:
I have made the changes.